### ISSUE TITLE
Segfault on video fullscreen

### ISSUE THREAD

**Issue by SomeAspy** [smartfrigde/armcord#619]
**Describe the bug**
Armcord will segfault when video content is fullscreened.
```
...200 more
[121706:0614/234120.995136:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= YUV_420_BIPLANAR and usage=SCANOUT_CPU_READ_WRITE
[121706:0614/234121.024551:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= YUV_420_BIPLANAR and usage=SCANOUT_CPU_READ_WRITE
[121706:0614/234121.059830:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= YUV_420_BIPLANAR and usage=SCANOUT_CPU_READ_WRITE
[121706:0614/234121.095111:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= YUV_420_BIPLANAR and usage=SCANOUT_CPU_READ_WRITE
[121706:0614/234121.124497:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= YUV_420_BIPLANAR and usage=SCANOUT_CPU_READ_WRITE
[121706:0614/234121.159787:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= YUV_420_BIPLANAR and usage=SCANOUT_CPU_READ_WRITE
[121706:0614/234121.195070:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= YUV_420_BIPLANAR and usage=SCANOUT_CPU_READ_WRITE
[121706:0100/000000.445254:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= YUV_420_BIPLANAR and usage=SCANOUT_CPU_READ_WRITE
# video set to fullscreen
/usr/bin/armcord: line 14: 121660 Segmentation fault      (core dumped) "$electron" /usr/share/armcord/app.asar $USER_FLAGS "$@"
```

**To Reproduce**
1. play media
2. click fullscreen

**Desktop (please complete the following information):**
 - OS: Arch (Zen Kernel) Wayland KDE
 - Method of installation: AUR
 - Version: [10b7e63](https://github.com/ArmCord/ArmCord/commit/10b7e638de7e80d442de63661169810837254ef6)

**Comment by SomeAspy** [comment]
Not reproducible in electron30. Possible upstream issue.
Workaround: edit `/usr/bin/armcord` to use `electron30` instead of `electron`

**Comment by francoism90** [comment]
https://issues.chromium.org/issues/331796411

I was just browsing if someone had the same issue, but it seems also to be in electron?

**Comment by SomeAspy** [comment]
this appears to be fixed in electron 31.1.0. The electron requirement has been updated in [6f1b8f4](https://github.com/ArmCord/ArmCord/commit/6f1b8f4c0aa02e9693dbf0b413c1020783bfe662)
